code/modules/surgery/organs/eyes.dm

```diff
@@ -167,6 +167,38 @@
 	emp_proof = TRUE
 	origin_tech = "materials=6;programming=5;biotech=6;magnets=6;syndicate=3"
 
+/obj/item/organ/internal/eyes/cybernetic/scope
+	name = "\improper Kaleido Optics eyes"
+	desc = "These cybernetic eye implants will let you zoom in on far away objects. Many users find it disorienting, and find it hard to interact with things near them when active."
```

Comment:
It feels weird to be setting the same description in two places, but the easiest way I see to avoid that would be having Initialize set the object's desc to the datum's, which also seems bleh.

Reply:
Out of scope (ha) reworking how rnd design datums work for description